api/src/main/java/org/apache/cloudstack/api/response/UserVmResponse.java

```diff
@@ -209,7 +209,7 @@
     private Long memoryKBs;
 
     @SerializedName("memoryintfreekbs")
-    @Param(description = "the internal memory thats free in vm")
+    @Param(description = "the internal memory thats free in vm or zero if it can not be calculated")
```

Comment:
Can we just not set it instead of 0, in the API response?

Reply:
It can be done but then it'll depend on the order in which the memory, free memory, is set, and any changes later on will break it

Reply:
```suggestion
    @Param(description = "the internal memory that's free in vm or zero if it can not be calculated")
```
sorry for this one @davidjumani